.env.example

```diff
@@ -0,0 +1 @@
+API_ENDPOINT=
```

Comment:
is it not supposed to have any value?

Reply:
This is .env.example. It is only meant to indicate what are the .env variables to run the app. 
And pushing .env with secrets is a big no no :fearful: